src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/NewHandler.java

```diff
@@ -90,7 +90,8 @@ protected IndentLevel getIndentImpl() {
             result = super.getIndentImpl();
 
             final boolean isLineWrappedNew = TokenUtil.isOfType(mainAst.getParent().getParent(),
-                                        TokenTypes.ASSIGN, TokenTypes.LITERAL_RETURN);
+                                        TokenTypes.ASSIGN, TokenTypes.COLON, TokenTypes.QUESTION,
```

Comment:
we might not need COLON, here and down as well.

Reply:
Rebase is a command, that allows to join two branch into one

please read this docs https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase